docs/guides/integration-mobile-overview.md

```diff
@@ -58,6 +49,71 @@ For details, refer to one of the following guides:
+| You want to use Prebid Mobile SDK and Prebid Server to send out ad requests from your iOS app | [UID2 GMA Plugin for iOS Integration Guide](../guides/mobile-plugin-ima-ios.md) |
+| You want to use Google GMA to integrate video, banner, interstitial, or native ads into your Android app | Both of the following, in this sequence:<ol><li>[UID2 GMA Plugin for Android Integration Guide](../guides/mobile-plugin-gma-android.md)</li><li>[Optional: UID2 GMA/IMA Plugin for GAM Secure Signals integration](../guides/integration-mobile-client-side.md#optional-uid2-gmaima-plugin-for-gam-secure-signals-integration)</li></ol> |
+| You want to use Google GMA to integrate video, banner, interstitial, or native ads into your iOS app | Both of the following, in this sequence:<ol><li>[UID2 GMA Plugin for iOS Integration Guide](../guides/mobile-plugin-gma-ios.md)</li><li>[Optional: UID2 GMA/IMA Plugin for GAM Secure Signals integration](../guides/integration-mobile-client-side.md#optional-uid2-gmaima-plugin-for-gam-secure-signals-integration)</li></ol> |
+| You want to use Google IMA to integrate multimedia ads into your Android app | Both of the following, in this sequence:<ol><li>[UID2 IMA Plugin for Android Integration Guide](../guides/mobile-plugin-ima-android.md)</li><li>[Optional: UID2 GMA/IMA Plugin for GAM Secure Signals integration](../guides/integration-mobile-client-side.md#optional-uid2-gmaima-plugin-for-gam-secure-signals-integration)</li></ol> |
+| You want to use Google IMA to integrate multimedia ads into your iOS app | Both of the following, in this sequence:<ol><li>[UID2 IMA Plugin for iOS Integration Guide](../guides/mobile-plugin-ima-ios.md)</li><li>[Optional: UID2 GMA/IMA Plugin for GAM Secure Signals integration](../guides/integration-mobile-client-side.md#optional-uid2-gmaima-plugin-for-gam-secure-signals-integration)</li></ol> |
+
+### Functionality Summary
+
+The following table summarizes the functionality available with the various integration options.
+
+| Implementation Option | Client-Side or Server-Side | Generate UID2 Token from DII | Refresh UID2 Token | Android/iOS? | Send Token to Bidstream |
+| :--- | :--- |  :--- | :--- | :--- | :--- |
+| UID2 SDK for Android | Either | &#9989; | &#9989; | Android | &ast; |
+| UID2 SDK for iOS | Either | &#9989;| &#9989; | iOS | &ast; |
+| GMA Plugin for Android<br/>(Requires&nbsp;Android&nbsp;SDK) | Server-Side | &#8212; | &#8212; | Android | &#9989;&ast;&ast; |
+| GMA Plugin for iOS<br/>(Requires iOS SDK) | Server-Side | &#8212; | &#8212; | iOS | &#9989;&ast;&ast; |
+| IMA Plugin for Android <br/>(Requires Android SDK)| Server-Side | &#8212; | &#8212; | Android | &#9989;&ast;&ast; |
+| IMA Plugin for iOS<br/>(Requires iOS SDK) | Server-Side | &#8212; | &#8212; | iOS | &#9989;&ast;&ast; |
+| Prebid Mobile SDK | Either | &#9989; | &#9989; | Either | &#9989;&ast;&ast;&ast; |
```

Comment:
Generate UID2 Token from DII and Refresh UID2 Token column should be "-" not green ticks.

Reply:
improvement suggestion: We shouldn't call this Prebid Mobile SDK - more like _Plugin for Prebid Mobile_ - we are saying we have a plugin that will make UID2 SDK works easily with Prebid Mobile SDK. 

Maybe the simplest fix is to rename to _UID2 <=> Prebid Mobile SDK Integration_?

Or in our [existing ](https://unifiedid.com/docs/guides/integration-mobile-client-side#optional-uid2-prebid-mobile-sdk-integration)Prebid Mobile SDK Integration doc, maybe we can rename the section to _Optional: Plugin for Prebid Mobile SDK Integration_ ?